jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/query/mapper/UserTaskInstanceQueryMapper.java

```diff
@@ -77,7 +77,10 @@ protected UserTaskInstanceDesc buildInstance(DataSet dataSetResult, int index) {
                 getColumnStringValue(dataSetResult, COLUMN_TASK_PROCESSID, index),//processId,
                 getColumnLongValue(dataSetResult, COLUMN_TASK_PROCESSINSTANCEID, index),//processInstanceId,
                 getColumnDateValue(dataSetResult, COLUMN_CREATEDON, index),//createdOn,
-                getColumnDateValue(dataSetResult, COLUMN_DUEDATE, index)//dueDate
+                getColumnDateValue(dataSetResult, COLUMN_DUEDATE, index),//dueDate
+                getColumnLongValue(dataSetResult, COLUMN_WORKITEMID, index),//workItemId
+                getColumnDateValue(dataSetResult, COLUMN_TASK_SLA_DUE_DATE, index),//slaCompliance
```

Comment:
I believe the comments for this line and the one below are swapped.

Reply:
@jstastny-cz yes, updated!